pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/PersistentTopics.java

```diff
@@ -311,6 +311,33 @@
      * @return a future that can be used to track when the topic is deleted
      */
     CompletableFuture<Void> deleteAsync(String destination);
+    
+    /**
+     * Unload a topic.
+     * <p>
+     *
+     * @param destination
+     *            Destination name
+     *
+     * @throws NotAuthorizedException
+     *             Don't have admin permission
+     * @throws NotFoundException
+     *             Destination does not exist
+     * @throws PulsarAdminException
+     *             Unexpected error
+     */
+    void unload(String destination) throws PulsarAdminException;
+
+    /**
+     * Unload a topic asynchronously.
+     * <p>
+     *
+     * @param destination
+     *            Destination name
+     *
+     * @return a future that can be used to track when the topic is unloaded
+     */
+    CompletableFuture<Void> unloadAsync(String destination);
```

Comment:
Should we start providing just async versions and expect users to just call a get on the future to get blocking versions.

Reply:
actually, this api is also used by cli-tool which requires blocking call. I think that's the reason why we have multiple api.